src/config.h.in

```diff
@@ -0,0 +1,4 @@
+#if XAPIAN_MAJOR_VERSION >= 2 || XAPIAN_MINOR_VERSION >=3 || (XAPIAN_MINOR_VERSION == 3 && XAPIAN_REVISION >= 6)
```

Comment:
Yes, added a fixup with that macro.

On Fri, Aug 12, 2016 at 12:55 AM, James Aylett <notifications@github.com>
wrote:

> In src/config.h.in
> <https://github.com/amandaJayanetti/RXapian/pull/25#discussion_r74486846>:
>
> > @@ -0,0 +1,4 @@
> > +#if XAPIAN_MAJOR_VERSION >= 2 || XAPIAN_MINOR_VERSION >=3 || (XAPIAN_MINOR_VERSION == 3 && XAPIAN_REVISION >= 6)
>
> Almost. You also need to delete src/config.h.in as it's no longer used,
> drop it as AC_CONFIG_HEADER() in configure.ac, and I don't think
> XAPIAN_OP_MAX_SUPPORT is defined anywhere at the moment?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/amandaJayanetti/RXapian/pull/25/files/ef9626298d83c1842f1d45f0a724bb461f304a7e#r74486846>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AH_jNi4VbjYyc1QMHwDckyBixNaSIiOdks5qe3cmgaJpZM4Jf4Xo>
> .
>